driver_go18_test.go

```diff
@@ -98,6 +106,118 @@ func getQueryProcess(processes []dbProcess, query string) *dbProcess {
 	return nil
 }
 
+func pollQueryProcess(db *sql.DB, dbName, query string, totalTimeout, pollTimeout time.Duration) (*dbProcess, error) {
+	end := time.Now().Add(totalTimeout)
+	var (
+		err error
+		processList []dbProcess
+		longProcess *dbProcess
+	)
+	for time.Now().Before(end) {
+		processList, err = getProcesslist(dbName, db)
+		longProcess = getQueryProcess(processList, query)
+
+		if err != nil || longProcess == nil {
+			time.Sleep(pollTimeout)
```

Comment:
you can use hardcoded timeout here, since it is just test. additional arguments makes it more complicated.

Reply:
removed